src/NServiceBus.ContainerTests/When_registering_components.cs

```diff
@@ -131,42 +97,6 @@ public void Setter_dependencies_should_be_supported_when_resolving_interfaces()
             }
         }
 
-
-        [Test]
-        public void Setter_dependencies_should_be_supported()
```

Comment:
Do you really think we can delete these tests? 

Removing these tests means we are not only no longer officially supporting `ConfigureProperty` but also we no longer officially support property setter injection in general. So that means if for some reason a container forgets to support setter injection existing code like

```
MyHandler : IHandleMessages
{
   public IFooService { get; set; }
}
```

will break. The milder form would be to rewrite the test to:


```
 [Test]
 -        public void Setter_dependencies_should_be_supported()
              Write Preview
Do you really think we can delete these tests? 

Removing these tests means we are not only no longer officially supporting `ConfigureProperty` but also we no longer officially support property setter injection in general. So that means if for some reason a container forgets to support setter injection existing code like

```
MyHandler : IHandleMessages
{
   public IFooService { get; set; }
}
```

will break. The milder form would be to rewrite the test two:



Do you really think we can delete these tests? 

Removing these tests means we are not only no longer officially supporting `ConfigureProperty` but also we no longer officially support property setter injection in general. So that means if for some reason a container forgets to support setter injection existing code like

```
MyHandler : IHandleMessages
{
   public IFooService { get; set; }
}
```

will break. The milder form would be to rewrite the test to just remove the `ConfigureProperty` calls and the corresponding Asserts.